Rakefile

```diff
@@ -25,8 +25,12 @@ if defined?(RSpec) && defined?(RSpec::Core::RakeTask)
   end
 end
 
-require 'jasmine'
-load 'jasmine/tasks/jasmine.rake'
+# Only load the jasmine tasks if we are within this repo, otherwise, the bundle
+# won't contain the jasmine gem (i.e., from manageiq)
+if ENV["BUNDLE_GEMFILE"].nil? || ENV["BUNDLE_GEMFILE"] == File.expand_path("../Gemfile", __FILE__)
```

Comment:
Also... `ENV["BUNDLE_GEMFILE"] == File.expand_path("../Gemfile", __FILE__)` doesn't look right.

Doesn't it only work when you have ui-classic in `manageiq/plugins/`. Nobody here does..